raw_data/preprocessed/africa/sierra_leone_english.txt

```diff
@@ -0,0 +1,132 @@
+
+Power to Delegate.
+(10) [1] Subject to subsection [2], the Minister may delegate any of his powers and responsibilities under this Act to the Director or such officer within the Ministry as he may think fit. [2] The Minister shall not delegate his power to make regulations under section 41.
+
+Department of the Environment.
+(11) [1] There is hereby established within the Ministry the Department of the Environment. [2] The Department shall be headed by the Director who shall be a public officer. [3] The Director shall be professionally knowledgeable in environmental protection matters. 
+
+Responsibilities of the Department.
+(12) [1] The Department of the Environment shall, subject to this this Act, coordinate all the environmentally related activities of Government Ministries and local authorities and act as the focal point of all national and international environmental matters relating to Sierra Leone. [2] In addition to its duties under subsection [1], the Department shall: [a] promote the formulation of national environmental goals and strategies; [b] formulate or promote the formulation of, and monitor the implementation of environmental Policies, programmes and projects, standards and regulations; [c] advise the Minister on environmental matters; [d] establish and coordinate working groups to study specific environmental problems or issues; [e] promote and support environmental education and training; [f] participate at crucial stages of the government's review of development programmes through environmental impact assessment reports; [g] prepare and publish overall assessment of the state of the environment; [h] coordinate and monitor the implementation of national environmental policies; [i] cooperate with Government Ministries, local authorities and other agencies on matters relating to environmental protection and management; [j] collect and make available to the public or interested persons or organisations, through publications and other appropriate means and in cooperation with public or private organizations, environmental data and information; [k] promote the establishment of national environmental standards; [l] provide secretarial services to the Board and other subsidiary bodies; [m] carry out other duties as are necessary or expedient for the full discharge of its duties under this Act.
```

Comment:
Please replace `this this` with `this`. Also, please replace `Policies` with `policies`.